### PR TITLE
Updating packageset version(s)

### PR DESCRIPTION
**Pull request by besirkurtulmus**
Renaming `allennlp==0.8.1` to `allennlp==0.8` according to PR [here](https://github.com/algorithmiaio/stagetools/pull/130).

Also added the `tensorflow-gpu==2.0` packageset in anticipation.

### CODE REVIEW

File: libraries/tensorflow-gpu-2.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+RUN pip install keras==2.1.4
```
Comment (besirkurtulmus) [inline libraries/tensorflow-gpu-2.0/Dockerfile]: @zeryx do you remember why we added `keras` to the dockerfile?

File: templates/tensorflow-gpu-2.0/src/__ALGO__.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import tensorflow.keras.backend as K
```
Comment (besirkurtulmus) [inline templates/tensorflow-gpu-2.0/src/__ALGO__.py]: I tested this algorithm template in the nightly build on CPU mode, and it worked!

File: libraries/tensorflow-gpu-2.0/Dockerfile (hunk 1 above)
Comment (zeryx) [reply libraries/tensorflow-gpu-2.0/Dockerfile]: yeah, version control - keras wildcards which version of tensorflow to install which leads to incompatibility.

**Comment by besirkurtulmus** [comment]
@kennydaniel just confirmed that the version range does work as intended in `requirements.txt`

**Pull request merged**